### PR TITLE
Adding getting started link

### PR DESCRIPTION
**Pull request by jeffbaumes**
Added a central button here:

screen shot 2015-10-19 at 2 06 53 pm

Which takes you to this page:

screen shot 2015-10-19 at 2 07 00 pm

@cpatrick @aashish24 does this look ok? This is bare-bones for now but the page needs something concrete to get people going with Resonant.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -846,6 +847,64 @@ <h3 class="section-subheading text-muted">Resonant's rich data visualization cap
...
+                            <!-- Project Details Go Here -->
+                            <h2>Getting Started</h2>
+                            <p class="portfolio-description">
+                                Depending on your development focus, you may want to start at various entry points in
+                                Resonant. As you need additional functionality, you can branch out and integrate with more
+                                of the Resonant components.
+                            </p>
+                            <h3>Data Management</h3>
+                            <p class="portfolio-description">
+                                Our full-featured data management platform, Girder, is a great place to start. Follow the
```
Comment (zachmullen) [inline index.html]: > full-featured

This phrase seems vague to me -- could we come up with a more descriptive adjective?

Comment (zachmullen) [inline index.html]: Suggestions: versatile, scalable, extensible, secure

Comment (jeffbaumes) [inline index.html]: @zachmullen I like versatile, I changed it.

**Comment by jeffbaumes** [comment]
@sobieavila look ok to you?

**Comment by sobieavila** [comment]
Yes. We still need to think about the target audience and maybe tailor those three buttons a bit more. Something like:

Developers: Get Started \* Custom Solutions \* Contact Us  

The first button would go to your newly created page, the second would go to a yet to be created page that would talk about specifically what we could do for customers who aren't developers  (who want to hire us to create a solution for them). And the third is our contact info as usual.

**Comment by jeffbaumes** [comment]
@sobieavila I like this approach. I've updated the button names to match:

screen shot 2015-10-20 at 2 53 49 pm

Currently Custom Solutions goes to our standard consulting page, but I'll open an issue to create a custom Custom Solutions page. The buttons are a but unbalanced but I think it's ok for now. Does this plan seem ok?

**Comment by sobieavila** [comment]
Looks good. I can't think of anything much shorter for the first button but I think the uneven size is OK (better to have unevenly-sized buttons with a clear message I think). 

Lisa

**Comment by jeffbaumes** [comment]
Ok thanks. I'll merge this and I opened #20.

**Pull request merged**